### DISCUSSION TITLE
Is there a "set mark" feature, like in emacs?

### DISCUSSION THREAD

**Discussion by hbd** [helix-editor/helix#3930] [Q&A] [answered]
Emacs has a "set mark" feature, i.e. shortcut `ctrl-space` by default, which is a text select based on setting the cursor in one place, then moving around to select the rest of the body of text.

Does helix have this feature? I have not been able to find reference to it in the docs.

![CleanShot 2022-09-21 at 13 38 29](https://user-images.githubusercontent.com/6836886/191605864-9c633e79-e0bf-43db-939a-c258c8124443.gif)

**Comment by kirawi** [comment, marked as answer]
Selection mode? `v`

**Reply by hbd** [reply]
This is exactly it. Thanks :)
